example/server/liquid_servlet.rb

```diff
@@ -10,20 +9,20 @@ def do_POST(req, res)
 
   private
 
-  def handle(type, req, res)
+  def handle(_type, req, res)
     @request = req
     @response = res
 
     @request.path_info =~ /(\w+)\z/
-    @action = $1 || 'index'
+    @action = Regexp.last_match(1) || 'index'
```

Comment:
I tend to disable that cop, because honestly this is just as bad if not worse. At least people are familiar with `$1` this not so much.